tests/actions/SessionTest.js

```diff
@@ -37,26 +53,38 @@ test('Authenticate is called with saved credentials when a session expires', ()
             const authenticateMock = jest.fn(() => Promise.resolve({
                 jsonCode: 200,
                 accountID: 1,
-                authToken: '12345',
+                authToken: TEST_AUTH_TOKEN,
                 email: TEST_USER_LOGIN,
             }));
             API.setMockCommand('Authenticate', authenticateMock);
+
+            // There's no need to use a jest.fn() here since we do not need to test whether this was called
+            // but must provide this response so the credentials are set after calling signIn()
             API.setMockCommand('CreateLogin', () => Promise.resolve({
                 jsonCode: 200,
                 accountID: 1,
-                authToken: '12345',
+                authToken: TEST_AUTH_TOKEN,
                 email: TEST_USER_LOGIN,
             }));
             signIn('Password1');
             expect(authenticateMock.mock.calls.length).toBe(1);
             return waitForPromisesToResolve();
         })
         .then(() => {
+            // Verify that our credentials were saved and that our session data is correct
+            expect(credentials.login).toBe(TEST_USER_LOGIN);
+            expect(credentials.autoGeneratedLogin).not.toBeUndefined();
+            expect(credentials.autoGeneratedPassword).not.toBeUndefined();
+            expect(session.authToken).toBe(TEST_AUTH_TOKEN);
+            expect(session.accountID).toBe(1);
```

Comment:
Creating a const for `TEST_AUTH_TOKEN` was a nice improvement. Any reason why the same shouldn't be done with `TEST_ACCOUNT_ID`?

Reply:
*not a blocker

Reply:
No reason. Thanks for pointing this out.